.github/actions/image-push/action.yml

```diff
@@ -0,0 +1,67 @@
+name: "Version hierarchy image pusher"
+description: "Will re-tag the docker image to create images for parent versions. For example, if the image is tagged 1.2.3, it will create a tag for 1.2 and 1.0 as well."
+
+inputs:
+  app_name:
+    description: "The name of the application."
+    required: true
+  package:
+    required: true
+  platform_name:
```

Comment:
My linter is suggesting to add `description` property to `package` and `platform_name`